package.json

```diff
@@ -7,9 +7,11 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.2",
     "@testing-library/user-event": "^7.1.2",
+    "lodash.find": "^4.6.0",
```

Comment:
please use lodash and import needed functions

Reply:
done